nix/ghc.nix

```diff
@@ -24,7 +24,6 @@ let
 
       # We need newer versions that build with GHC 8.6
       language-c = super.callPackage ./overrides/language-c-0.8.2.nix {};
```

Comment:
We can probably also get rid of the `language-c` override if we don’t need `c2hs`.

Reply:
I believe we need it for grpc related stuff.

Reply:
The Nix override seems to be unused. I could only find references to the Hazel package.

Reply:
:+1: